src/lib/containers/entity/annotations/AdditionalPropertiesSchemaField.tsx

```diff
@@ -73,14 +87,13 @@ export function getWidgetFromPropertyType(
   propertyType: PropertyType,
 ): AdditionalPropertyWidget {
   switch (propertyType) {
-    case PropertyType.INTEGER:
-    case PropertyType.FLOAT:
-      return AdditionalPropertyWidget.UpDownWidget
```

Comment:
Don't need to use this UpDownWidget anymore because we're using the (custom) TextWidget for all numbers and strings